.github/workflows/publish.yml

```diff
@@ -57,33 +58,6 @@ jobs:
           echo -e "COMMIT_URL = ${COMMIT_URL}"
           echo -e "COMMITTER = ${COMMITTER}"
 
-      - name: Define target branches in wireapp/wire-builds to bump
-        id: output_target_branches
-        shell: bash
-        run: |
-          wire_builds_target_branches='[]'
-
-          version_tag="${TAG:-$BRANCH_NAME}"
-
-          if [[ "$version_tag" == *"production"* ]]; then
-            echo "FUTUREWORK: bump some production branch on wire-builds once it exists"
-          fi
-
-          if [[ "$version_tag" == *"staging"* ]]; then
-            echo "FUTUREWORK: bump some cloud staging branch on wire-builds once it exists"
-          fi
-
-          if [ "$version_tag" == "dev" ]; then
-            wire_builds_target_branches='["dev"]'
-          fi
-
-          if [ "$version_tag" == "release/q1-2024" ]; then
-            wire_builds_target_branches='["q1-2024"]'
-          fi
-
-          echo "wire_builds_target_branches: $wire_builds_target_branches"
-          echo "wire_builds_target_branches=$wire_builds_target_branches" >> $GITHUB_OUTPUT
```

Comment:
This was simplified and moved later on in the job (see https://github.com/wireapp/wire-webapp/pull/17006/files#diff-551d1fcf87f78cc3bc18a7b332a4dc5d8773a512062df881c5aba28a6f5c48d7R160)